drivers/scd30/scd30_saul.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2020 Technische Universität Braunschweig
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_scd30
+ * @file
+ * @brief       SAUL adaption for Sensirion SCD30 sensor
+ * @author      Puhang Ding      <czarsir@gmail.com>
+ * @author		Nishchay Agrawal <f2016088@pilani.bits-pilani.ac.in>
+ * @author      Jan Schlichter   <schlichter@ibr.cs.tu-bs.de>
+ */
+
+#include "saul.h"
+#include "scd30.h"
+#include "scd30_internal.h"
+
+static void _float_fit(float src, phydat_t *data, uint32_t mul)
+{
+    int32_t i32;
+
+    i32 = src * mul;
+    phydat_fit(data, &i32, 1);
+}
+
+static int _co2_read(const void *dev, phydat_t *res)
+{
+    int ret = 0;
+    scd30_measurement_t result;
+
+    scd30_set_param(dev, SCD30_START, SCD30_DEF_PRESSURE);
+
+    ret = scd30_read_triggered((scd30_t *)dev, &result);
+    if (ret != 0) {
+        puts("[scd30_saul] Error reading co2 concentration\n");
```

Comment:
You'll need to include `<stdio.h>` for that.
Or better just drop it - you are already returning an error code and this is not adding any new information.

I just checked a couple of other drivers and none of them are verbose.

Reply:
Yeah sure, removed them